src/frontend/app/components/ScenarioContent.tsx

```diff
@@ -31,6 +32,7 @@ export default function ScenarioContent({ scenario }: { scenario: Scenario }) {
         </div>
         <p className="text-2xl leading-9 mb-4">{scenario.summary}</p>
         <TagList scenario={scenario} />
+        {!!scenario.integrations.results.length && <ScenarioIntegrations scenario={scenario} />}
```

Comment:
Minor - it would be great to move this logic inside on <ScenarioIntegrations> and keep the markup cleaner.